src/status_im/contexts/wallet/collectible/events.cljs

```diff
@@ -131,6 +137,50 @@
               :account    current-viewing-account
               :amount     collectibles-request-batch-size}]]]})))
 
+(rf/reg-event-fx
+ :wallet/collectible-ownership-update-finished-with-error
+ (fn [{:keys [db]} [{:keys [accounts message chainId]}]]
+   (let [address   (first accounts)
+         chain-ids (get-in db [:wallet :ui :collectibles :fetching address])]
+     (log/error "[wallet] collectibles ownership update failed"
+                {:address  address
+                 :chain-id chainId
+                 :error    message})
+     {:db (cond-> db
+            (some? chain-ids)
+            (update-in [:wallet :ui :collectibles :fetching address] disj chainId))})))
+
+(rf/reg-event-fx
+ :wallet/collectible-ownership-update-finished
+ (fn [{:keys [db]} [{:keys [accounts message chainId]}]]
+   (let [address          (first accounts)
+         {:keys [added]}  (transforms/json->clj message)
+         added?           (some? added)
+         chain-ids        (some-> (get-in db [:wallet :ui :collectibles :fetching address])
+                                  (disj chainId))
+         pending-requests (get-in db [:wallet :ui :collectibles :pending-requests] 0)]
+     {:db (cond-> db
+            (empty? chain-ids)
+            (update-in [:wallet :ui :collectibles :fetching] dissoc address)
+
+            (seq chain-ids)
+            (assoc-in [:wallet :ui :collectibles :fetching address] chain-ids)
+
+            added?
+            (assoc-in [:wallet :ui :collectibles :pending-requests] (inc pending-requests)))
+      :fx [(when added?
+             [:dispatch
+              [:wallet/request-new-collectibles-for-account
+               {:request-id (-> (get-unique-collectible-request-id 1) first)
+                :account    address
+                :chain-ids  [chainId]
+                :amount     collectibles-request-batch-size}]])]})))
+
+(rf/reg-event-fx
+ :wallet/set-collectibles-fetching-status
+ (fn [{:keys [db]} [address chain-ids]]
+   {:db (assoc-in db [:wallet :ui :collectibles :fetching address] chain-ids)}))
```

Comment:
Setting the chain-ids as the value instead of the boolean value to know which chains are been fetched. Later this can be used to show any loading indicator or info box that collectibles are loading.